### PR TITLE
Update GenOuest config

### PR DESCRIPTION
**Pull request by abretaud**
Updating the GenOuest profile (added new default mounted volume + updated cpu/mem limits)

### CODE REVIEW

**Comment by abretaud** [comment]
ping @nf-core/configs-team for a little review pleae :)

**Comment by jfy133** [comment]
Sorry @abretaud ! We replaced configs team with nf-core/maintainers  but forgot to update the links here! Looking nnow!

File: docs/genouest.md [hunk 1]
```diff
@@ -8,10 +8,10 @@ To use, run the pipeline with `-profile genouest`. This will download and launch
 
 Nextflow is installed on the GenOuest cluster. Some documentation is available on the [GenOuest website](https://www.genouest.org/howto/#nextflow).
 
-You need to activate it like this:
+You need to activate it like this (or any more recent version in the same directory):
 
 ```bash
-source /local/env/envnextflow-19.07.0.sh
+source /local/env/envnextflow-22.10.4.sh
```
Comment (jfy133) [inline docs/genouest.md:14]: Note many pipelines use a more recent minimum version

**Review by jfy133 — APPROVED** [review]
LGTM if no `docs.md` info needs to be updated :)

**Pull request merged**